backend/config/boltdb.go

```diff
@@ -0,0 +1,90 @@
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/boltdb/bolt"
+)
+
+type BoltReaderWriterQuerierImpl struct {
+	bucket  string
+	version string
+	db      *bolt.DB
+}
+
+func NewBoltReader(version string, db *bolt.DB) Reader {
```

Comment:
we Should not duplicate New Function like this

Reply:
We can fix this by create an interface compose (reader, writer, Querier) and return that interface to prevent dupplicate